lms/static/scripts/frontend_apps/components/dashboard/test/OrganizationActivityFilters-test.js

```diff
@@ -0,0 +1,272 @@
+import { MultiSelect } from '@hypothesis/frontend-shared';
+import {
+  checkAccessibility,
+  mockImportedComponents,
+} from '@hypothesis/frontend-testing';
+import { mount } from 'enzyme';
+import sinon from 'sinon';
+
+import { Config } from '../../../config';
+import OrganizationActivityFilters, {
+  $imports,
+} from '../OrganizationActivityFilters';
+
+describe('OrganizationActivityFilters', () => {
+  const courses = [
+    {
+      id: 1,
+      title: 'Course A',
+    },
+    {
+      id: 2,
+      title: 'Course B',
+    },
+  ];
+  const assignments = [
+    {
+      id: 1,
+      title: 'Assignment 1',
+    },
+    {
+      id: 2,
+      title: 'Assignment 2',
+    },
+  ];
+  const students = [
+    {
+      lms_id: '1',
+      display_name: 'First student',
+    },
+    {
+      lms_id: '2',
+      display_name: 'Second student',
+    },
+  ];
+
+  let fakeUseAPIFetch;
+  let fakeConfig;
+  let onCoursesChange;
+  let onAssignmentsChange;
+  let onStudentsChange;
+  let wrappers = [];
+
+  /**
+   * @param {boolean} loading
+   */
+  function configureFakeAPIFetch(fakeUseAPIFetch, loading) {
+    fakeUseAPIFetch.onCall(0).returns(
+      loading
+        ? { isLoading: true }
+        : {
+            isLoading: false,
+            data: { courses },
+          },
+    );
+    fakeUseAPIFetch.onCall(1).returns(
+      loading
+        ? { isLoading: true }
+        : {
+            isLoading: false,
+            data: { assignments },
+          },
+    );
+    fakeUseAPIFetch.onCall(2).returns(
+      loading
+        ? { isLoading: true }
+        : {
+            isLoading: false,
+            data: { students },
+          },
+    );
+  }
+
+  beforeEach(() => {
+    fakeUseAPIFetch = sinon.stub();
+    configureFakeAPIFetch(fakeUseAPIFetch, false);
+
+    onCoursesChange = sinon.stub();
+    onAssignmentsChange = sinon.stub();
+    onStudentsChange = sinon.stub();
+    wrappers = [];
+
+    fakeConfig = {
+      dashboard: {
+        routes: {
+          courses: '/api/dashboard/courses',
+          assignments: '/api/dashboard/assignments',
+          students: '/api/dashboard/students',
+        },
+      },
+    };
+
+    $imports.$mock(mockImportedComponents());
+    $imports.$mock({
+      '../../utils/api': {
+        useAPIFetch: fakeUseAPIFetch,
+      },
+    });
+  });
+
+  afterEach(() => {
+    wrappers.forEach(w => w.unmount());
+    $imports.$restore();
+  });
+
+  /**
+   @param {Object} [selection]
+   @param {Object[]} [selection.selectedStudents] - Currently selected students.
+   @param {Object[]} [selection.selectedAssignments] - Currently selected assignments.
+   @param {Object[]} [selection.selectedCourses] - Currently selected courses.
+   */
```

Comment:
I'm not sure this comment adds much value, since the types are not checked and the meaning seems quite clear from a glance at the implementation.